integration/src/test/scala/com/snowplowanalytics/snowplow/collectors/scalastream/intergation/KinesisSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.collectors.scalastream.intergation
+
+import cats.effect.{IO, Sync}
+import cats.effect.testing.specs2.CatsIO
+import com.snowplowanalytics.snowplow.collectors.scalastream.intergation.TestUtils.Http.Request.RequestType.{Bad, Good}
+import com.snowplowanalytics.snowplow.collectors.scalastream.intergation.TestUtils._
+import com.snowplowanalytics.snowplow.eventgen.tracker.HttpRequest.Method.Post
+import org.specs2.mutable.Specification
+
+class KinesisSpec extends Specification with CatsIO {
+  "The Kinesis collector should" >> {
+    "ensure all events are written to the sink" in {
+      val localstack = Containers.localstack
+      val collector  = Containers.collector("kinesis", "config", Some(localstack))
+
+      lazy val localstackPort = Containers.getExposedPort(localstack, 4566)
+      lazy val collectorPort  = Containers.getExposedPort(collector, 12345)
+
+      val resources = for {
+        localstack <- Containers.mkContainer[IO](localstack)
+        collector  <- Containers.mkContainer[IO](collector)
+        kinesis    <- Kinesis.mkKinesisClient[IO](localstackPort)
+        httpClient <- Http.mkHttpClient[IO]
+      } yield (localstack, collector, kinesis, httpClient)
```

Comment:
```suggestion
      } yield (kinesis, httpClient)
```
They are not used after.